modules/gdscript/gdscript.cpp

```diff
@@ -874,44 +874,46 @@ Variant GDScript::callp(const StringName &p_method, const Variant **p_args, int
 }
 
 bool GDScript::_get(const StringName &p_name, Variant &r_ret) const {
-	{
-		const GDScript *top = this;
-		while (top) {
-			{
-				HashMap<StringName, Variant>::ConstIterator E = top->constants.find(p_name);
-				if (E) {
-					r_ret = E->value;
-					return true;
-				}
+	if (p_name == GDScriptLanguage::get_singleton()->strings._script_source) {
+		r_ret = get_source_code();
+		return true;
+	}
+
+	const GDScript *top = this;
+	while (top) {
+		{
+			// Constants.
+			HashMap<StringName, Variant>::ConstIterator E = top->constants.find(p_name);
```

Comment:
This is a nitpick of mine but I don't think this type of comment says much, since in the very next line you can see it's about the constants. It does not harm though.